DevoxxClientMobile/src/main/java/com/devoxx/service/DevoxxService.java

```diff
@@ -969,22 +970,32 @@ private void loadCfpAccount(User user, Runnable successRunnable) {
             Services.get(SettingsService.class).ifPresent(settingsService -> {
                 String devoxxCfpAccountUuid = settingsService.retrieve(DevoxxSettings.SAVED_ACCOUNT_ID);
                 if (devoxxCfpAccountUuid == null) {
-                    RemoteFunctionObject fnVerifyAccount = RemoteFunctionBuilder.create("verifyAccount")
-                            .param("0", getConference().getCfpEndpoint())
-                            .param("1", user.getNetworkId())
-                            .param("2", user.getLoginMethod().name())
-                            .param("3", user.getEmail())
-                            .object();
-                    GluonObservableObject<String> accountUuid = fnVerifyAccount.call(String.class);
-                    accountUuid.initializedProperty().addListener((obs, ov, nv) -> {
-                        LOG.log(Level.INFO, "Verified user " + user + " as account with uuid " + accountUuid);
-                        cfpUserUuid.set(accountUuid.get());
-                        settingsService.store(DevoxxSettings.SAVED_ACCOUNT_ID, accountUuid.get());
+                    if (user.getLoginMethod() == LoginMethod.Type.CUSTOM) {
+                        LOG.log(Level.INFO, "Logged in user " + user + " as account with uuid " + user.getNetworkId());
+                        cfpUserUuid.set(user.getNetworkId());
+                        settingsService.store(DevoxxSettings.SAVED_ACCOUNT_ID, user.getNetworkId());
 
                         if (successRunnable != null) {
                             successRunnable.run();
                         }
-                    });
+                    } else {
+                        RemoteFunctionObject fnVerifyAccount = RemoteFunctionBuilder.create("verifyAccount")
+                                .param("0", getConference().getCfpEndpoint())
+                                .param("1", user.getNetworkId())
+                                .param("2", user.getLoginMethod().name())
+                                .param("3", user.getEmail())
+                                .object();
+                        GluonObservableObject<String> accountUuid = fnVerifyAccount.call(String.class);
+                        accountUuid.initializedProperty().addListener((obs, ov, nv) -> {
+                            LOG.log(Level.INFO, "Verified user " + user + " as account with uuid " + accountUuid);
```

Comment:
Doesn't hurt to check if the value of nv is true.

Reply:
Actually, the initialized property can only become `true`. But it doesn't hurt indeed.